src/data/bucket/fill_extrusion_bucket.js

```diff
@@ -588,6 +600,29 @@ function tileToMeter(canonical: CanonicalTileID) {
     return circumferenceAtEquator * 2 * exp / (exp * exp + 1) / EXTENT / (1 << canonical.z);
 }
 
+function isAOConcaveAngle(p2, p1, p3) {
+    if (p2.x < 0 || p2.x >= EXTENT || p1.x < 0 || p1.x >= EXTENT || p3.x < 0 || p3.x >= EXTENT) {
+        return false; // angles are not processed for edges that extend over tile borders
+    }
+    const a = p3.sub(p1);
+    const an = a.perp();
+    const b = p1.sub(p2);
```

Comment:
Could you define these vectors as `a = p3 - p1, b = p2 - p1`? Right now this is pretty confusing as you're not actually comparing the angle between two vectors but `pi - x` instead.

Reply:
Done.